src/components/CardEditor/CardEditForm/_card-edit-form.scss

```diff
@@ -115,12 +115,23 @@
     align-items: flex-end;
     padding-bottom: 1rem;
     &--item {
+      .#{$iot-prefix}--color-dropdown {
+        margin-bottom: 1.25rem;
+      }
       margin-right: 1rem;
       width: 100%;
     }
     &--item-half {
+      display: flex;
+      flex-direction: column;
       margin-right: 1rem;
       width: 50%;
+      &-label {
+        margin-bottom: 1rem;
+      }
+      &-content {
+        margin-bottom: 0.6rem;
```

Comment:
is it necessary to not use standard token size here?

Reply:
This was needed to line up the `read-only` form options with the dropdowns on the same horizontal line.